index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header id = "header">
+        <img id="header-img" src ="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" />
+        <nav id = "nav-bar"> 
+          <ul> 
+           <a href="#features"  class="nav-link"> <li> Features </li> </a> 
+            <a href="#howitworks" class="nav-link"> <li>How it Works </li> </a> 
+            <a href="#pricing" class="nav-link"> <li> Pricing </li> </a> 
+          </ul>
+           </nav>
+         </header>
+         <h2 id="features">
```

Comment:
Please ensure you use HTML5 semantic tags throughout your code. Ensure all the main content in your HTML file is wrapped in a <main> </main> tag and all code within is wrapped in the <article> </article> tag.

Reply:
- [ ] Ouch! this is close but please make sure your comments are clear and easy to understand because the code you've highlighted above uses the appropriate semantic tags and this will end up confusing the reviewee. 
- [ ] The last comment "...and all code within is wrapped in the tag." is a bit vague because you're not specifying which tag it should be wrapped in so please improve these comments. ✔